sos/help/__init__.py

```diff
@@ -206,7 +207,8 @@ def display_self_help(self):
             'report.plugins.$plugin': 'Information on a specific $plugin',
             'clean':    'Detailed help on the clean command',
             'collect':  'Detailed help on the collect command',
-            'policies': 'How sos operates on different distributions'
+            'policies': 'How sos operates on different distributions',
+            'upload': 'Detailed help on the upload command'
```

Comment:
I might swap ordering of these two lines, to have `upload` listed close to other `sos` subcommands. `policies` is bit extra here.

Reply:
Good idea, I'll do that.